spec/models/tag_spec.rb

```diff
@@ -76,7 +76,7 @@ def create_tags
       tags = create_tags
       fetched = GalleryGroup.where(id: tags.map(&:id)).select(:id).ordered_by_id.with_character_counts
       expect(fetched).to eq(tags)
-      expect(fetched.map{|x| x[:character_count] }).to eq([0, 1, 2])
+      expect(fetched.map{|x| x[:character_count] }).to eq([0, 1, 2]) # rubocop:disable Rails/Pluck
```

Comment:
Could use
```suggestion
      expect(fetched.to_a.pluck(:character_count)).to eq([0, 1, 2])
```
I guess? But it doesn't seem like it's much clearer imo